recipes/install.rb

```diff
@@ -65,7 +65,7 @@
 end
 
 user node['hopsworks']['user'] do
-  home "/home/#{node['hopsworks']['user']}"
+  home node['hopsworks']['user-home']
```

Comment:
Is this defined somewhere? I only see ['glassfish']['user-home'] up there

Reply:
Interesting, I wonder how it worked. Anyway I changed it to `node['glassfish']['user-home']`